OpenRA.Mods.Common/AI/AttackOrFleeFuzzy.cs

```diff
@@ -25,7 +25,7 @@ sealed class AttackOrFleeFuzzy
 		{
 			"if ((OwnHealth is Normal) " +
 			"and ((EnemyHealth is NearDead) or (EnemyHealth is Injured) or (EnemyHealth is Normal)) " +
-			"and ((RelativeAttackPower is Weak) or (RelativeAttackPower is Equal) or (RelativeAttackPower is Strong)) " +
+			"and ((RelAtkPwr is Weak) or (RelAtkPwr is Equal) or (RelAtkPwr is Strong)) " +
```

Comment:
I don't buy that these would make any measurable performance difference, and IMO the shortened versions are a lot less readable.  Can we please drop these renames?